README.md

```diff
@@ -59,27 +67,40 @@ This code is compiled into Wasm bytecode as part of the smart contract.
   storing, and caching wasm artifacts.
-- [go-cosmwasm](https://github.com/CosmWasm/go-cosmwasm) - High-level go
-  bindings to all the power inside `cosmwasm-vm`. Easily allows you to upload,
-  instantiate and execute contracts, making use of all the optimizations and
-  caching available inside `cosmwasm-vm`.
+- [wasmvm](https://github.com/CosmWasm/wasmvm) - High-level go bindings to all
+  the power inside `cosmwasm-vm`. Easily allows you to upload, instantiate and
+  execute contracts, making use of all the optimizations and caching available
+  inside `cosmwasm-vm`.
 - [wasmd](https://github.com/CosmWasm/wasmd) - A basic Cosmos SDK app to host
-  WebAssembly smart contracts.
-
-Ongoing work is currently tracked
-[on this project board](https://github.com/orgs/CosmWasm/projects/1) for all of
-the blockchain / contract work.
+  WebAssembly smart contracts. It can be run as is, or you can import the
+  `x/wasm` module from it and use it in your blockchain. It is designed to be
+  imported and customized for other blockchains, rather than forked.
 
 ## Creating a Smart Contract
 
 You can see some examples of contracts under the `contracts` directory, which
-you can look at.
-
-If you want to get started building you own, the simplest way is to go to the
-[cosmwasm-template](https://github.com/CosmWasm/cosmwasm-template) repository
-and follow the instructions. This will give you a simple contract along with
-tests, and a properly configured build environment. From there you can edit the
-code to add your desired logic and publish it as an independent repo.
+you can look at. They are simple and self-contained, primarily meant for testing
+purposes, but that also makes them easier to understand.
+
+You can also look at [cosmwasm-plus](https://github.com/CosmWasm/cosmwasm-plus)
+for examples and inspiration on more production-like contracts and also how we
+call one contract from another. If you are working on DeFi or Tokens, please
+look at the `cw20`, `cw721` and/or `cw1155` packages that define standard
+interfaces as analogues to some popular ERC designs. (`cw20` is also inspired by
+`erc777`).
+
+If you want to get started building you own contract, the simplest way is to go
+to the [cosmwasm-template](https://github.com/CosmWasm/cosmwasm-template)
+repository and follow the instructions. This will give you a simple contract
+along with tests, and a properly configured build environment. From there you
+can edit the code to add your desired logic and publish it as an independent
+repo.
+
+We also recommend you review our [documentation site](https://docs.cosmwasm.com)
+which contains a few tutorials to guide you in building your first contracts. We
+also do public workshops on various topics about once a month. You can find
+[past recordings under the "Videos" section](https://cosmwasm.com/resources), or
+[join our Discord server](https://docs.cosmwasm.com/chat) to ask for help.
 
 ## API entry points
```

Comment:
It would be best to move these implementation details to different folders, use `README.md` as an introduction and index to other docs.

Reply:
Simon, what do you think of that?
An idea of where to put such details? Do we still need them?

Reply:
The only audience for those is hardcore low level hackers and people who implement alternative standard libraries. No idea how to address them without exposing distracting detail to contract developers.